### PR TITLE
Updating Vocalink to v612 (12 JUNE 2020)

### PR DESCRIPTION
**Pull request by htmlandbacon**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -264,12 +264,11 @@ export default class UkModulusChecking {
    */
 
   loadValacdos() {
-    const content = fs.readFileSync(`${__dirname}/data/valacdos-v570-updated.txt`, 'utf8');
+    const content = fs.readFileSync(`${__dirname}/data/valacdos-v612-updated.txt`, 'utf8');
     const valacdos = [];
 
     content.split('\r\n').forEach((line) => {
       const data = line.split(/\s+/);
-
```
Comment (franciscocardoso) [inline src/index.js:272]: Can you please revert this change?

Comment (htmlandbacon) [reply src/index.js]: Have reverted, thanks!
